magicctapipe/scripts/mars/tests/labels_and_units_for_params_comparison.py

```diff
@@ -0,0 +1,145 @@
+import math
+
+mars_params = {
+    "event_id" : "MRawEvtHeader_1.fStereoEvtNumber",
+    "length_M1" : "MHillas_1.fLength", "length_M2" : "MHillas_2.fLength",
+    "width_M1" : "MHillas_1.fWidth", "width_M2" : "MHillas_2.fWidth",
+    "size_M1" : "MHillas_1.fSize", "size_M2" : "MHillas_2.fSize",
+    "slope_M1" : "MHillasTimeFit_1.fP1Grad", "slope_M2" : "MHillasTimeFit_2.fP1Grad",
+    "delta_M1" : "MHillas_1.fDelta", "delta_M2" : "MHillas_2.fDelta",
+    "cogx_M1" : "MHillas_1.fMeanX", "cogx_M2" : "MHillas_2.fMeanX",
+    "cogy_M1" : "MHillas_1.fMeanY", "cogy_M2" : "MHillas_2.fMeanY",
+    "hmax" : "MStereoPar.fMaxHeight",
+    "corex" : "MStereoPar.fCoreX",
+    "corey" : "MStereoPar.fCoreY",
+    "az" : "MStereoPar.fDirectionAz",
+    "zd" : "MStereoPar.fDirectionZd",
+    "impact_M1" : "MStereoPar.fM1Impact", "impact_M2" : "MStereoPar.fM2Impact",
+    "hmax_disp" : "MStereoParDisp.fMaxHeight",
+    "corex_disp" : "MStereoParDisp.fCoreX",
+    "corey_disp" : "MStereoParDisp.fCoreY",
+    "az_disp" : "MStereoParDisp.fDirectionAz",
+    "zd_disp" : "MStereoParDisp.fDirectionZd",
+    "impact_disp_M1" : "MStereoParDisp.fM1Impact", "impact_disp_M2" : "MStereoParDisp.fM2Impact",
+    # "n_islands" : "MImagePar_1./MImagePar_1.fNumIslands",
+    "leakage_M1" : "MNewImagePar_1./MNewImagePar_1.fLeakage1", "leakage_M2" : "MNewImagePar_2./MNewImagePar_2.fLeakage1"
+
+}
+mcp_params = {
+    "length_M1" : "length", "length_M2" : "length",
+    "width_M1" : "width", "width_M2" : "width",
+    "size_M1" : "intensity", "size_M2" : "intensity",
+    "slope_M1" : "slope", "slope_M2" : "slope",
+    "delta_M1" : "psi", "delta_M2" : "psi",
+    "cogx_M1" : "y", "cogx_M2" : "y",
+    "cogy_M1" : "x", "cogy_M2" : "x",
+    "hmax" : "h_max",
+    "corex" : "core_x",
+    "corey" : "core_y",
+    "az" : "az",
+    "zd" : "alt",
+    "impact_M1" : "impact", "impact_M2" : "impact",
+    "hmax_disp" : "h_max",
+    "corex_disp" : "core_x",
+    "corey_disp" : "core_y",
+    "az_disp" : "az",
+    "zd_disp" : "alt",
+    "impact_disp_M1" : "impact", "impact_disp_M2" : "impact",
+    # "n_islands" : "n_islands",
+    "leakage_M1" : "intensity_width_1", "leakage_M2" : "intensity_width_1"
+}
+
+scale_factors = {
+    "length_M1" : 1000. / 1.0713, "length_M2" : 1000. / 1.0713,
+    "width_M1" : 1000. / 1.0713, "width_M2" : 1000. / 1.0713,
+    "size_M1" : 1., "size_M2" : 1.,
+    "slope_M1" : 0.0033703*1.64*1.0713, "slope_M2" : 0.0033703*1.64*1.0713,
```

Comment:
I added the scaling for slope how I think it should be, but it actually does not work, and I am not sure why.
0.0033703/1.64/1.0713 would work, but it does not make sense to me why it should be like this

Reply:
Hi Julian, these factors are used to convert the units used in ctapipe into the MARS units. For the other parameters we also divide by e.g. 1.0713 so it would make sense to me, that it works if you divide instead of multiplying. But that is just a guess, I would need to check, since slope was the parameter that was still failing when we ran the tests.

Reply:
yes, but e.g. length and width are in MARS in units of mm, while slope is time slices / mm, so if you divide the first one it is expected that the second one should be multiplied. Either way this 7% effect is not big, and here it is difficult to say if it matches better or not. I meant more the 1.64 factor. MARS files have slope in time slices / mm, and arrival times are given in time slices in the images, but then ctapipe_io_magic converts them to ns by dividing by 1.64 [GHZ].  We had to do it, because there was a problem when comparing with sim_telarray MCs, so what gets out of MCP should be in ns/deg. Multiplication by 0.00337 is making it .../mm and this part seems to work (not sure about the 1.07 factor), but to make it back time slices / mm it would need to be multiplied again by 1.64 (cancelling the original division), and this does not seem to work.

Reply:
Hi Julian, are you sure you need the 0.00337 factor? As I see it, both hillas and timing parameters come out in unit of meters, and not degrees. They would be in degrees if the geometry is converted in the telescope frame (if I remember correctly, or camera one, now I am not sure) using the choice of focal length done at the EventSource level (i.e. nominal or effective), so that also the geometry units are degrees and all the parameters calculated starting from it.

Reply:
Sorry for a super late feedback on this. 
In the test that I did without the 0.00337 factor the slope was mismatching by a large factor. Maybe some other changes in the code affected it. Either way, at least 1.64 factor should be there, because MARS is using time slices and ctapipe ns. 
Are the tests working fine now for the slope parameter?